crates/cubecl-runtime/src/server.rs

```diff
@@ -125,6 +142,27 @@ impl Handle {
     }
 }
 
+/// Tensor representation with an owned [server handle](Handle),
+/// the strides and the shape.
+#[derive(new, Debug, Clone)]
+pub struct TensorHandle {
+    /// The handle to the memory resource
+    pub handle: Handle,
+    /// The strides of this buffer
+    pub strides: Vec<usize>,
+    /// The shape of this buffer
+    pub shape: Vec<usize>,
+    /// The size of each element
+    pub elem_size: usize,
```

Comment:
What is the difference between this and `TensorHandleRef` provided in `cubecl_linalg`?

I understand that this one is owned, but why do you need an owned version?